WcaOnRails/app/views/delegate_reports/edit.html.erb

```diff
@@ -11,7 +11,8 @@
     <% if @delegate_report.posted? %>
       <%= f.input :discussion_url %>
     <% end %>
-    <%= f.input :schedule_url %>
+    <% default_schedule_url = @competition.competition_venues.any? ? link_to_competition_schedule_tab(@competition) : '' %>
+    <%= f.input :schedule_url, input_html: { value: default_schedule_url } %>
```

Comment:
Given this forces WCA schedule URL, perhaps we should disable that field when using `default_schedule_url`?

Reply:
I'm a bit confused- are you saying that if a WCA schedule URL exists already we don't need to have this field and we should just automatically link to it?

Reply:
That feels reasonable to me. Are there situations where we'd want to let people enter something other than the WCA schedule here, if a WCA schedule exists?

Reply:
Ah sorry I missed your comment! My point was just that if there is WCA schedule and we fill this field, it's certainly the URL we want (so it makes sense to disable it).

Reply:
By "disable" do you mean use the disabled HTML attribute, or just omit the field entirely?

Reply:
I mean just disabling the element, although I imagine us getting rid of it in the future ^^

Reply:
I mean just disabling the element, although I imagine us getting rid of it in the future ^^

Reply:
```suggestion
    <%= f.input :schedule_url, input_html: { value: default_schedule_url }, disabled: default_schedule_url.present? %>
```

I think this should work out fine :)

Reply:
I can't commit to @UnsolvedCypher's repository, so to amend this we either need Matt to fix it or a brand new PR. I can take care of the latter if necessary :)